pom.xml

```
@@ -10,7 +10,7 @@
   </parent>
 
   <artifactId>browserstack-integration</artifactId>
-  <version>1.1.7-SNAPSHOT</version>
+  <version>1.1.8-SNAPSHOT</version>
```

```suggestion
  <version>1.1.9-SNAPSHOT</version>
```

As `1.1.8` is released.